codegen/smithy-kotlin-codegen/src/test/kotlin/software/amazon/smithy/kotlin/codegen/rendering/PaginatorGeneratorTest.kt

```diff
@@ -762,4 +762,308 @@ class PaginatorGeneratorTest {
 
         actual.shouldContainOnlyOnceWithDiff(expected)
     }
+
+    @Test
+    fun testRenderPaginatorWithDocumentList() {
```

Comment:
**Nit**: Let add a comment to each linking the issue that spawned them.

Reply:
I'll create one, we don't have an external issue for this

Reply:
Oh maybe not required if there wasn't an external issue...that's just busywork.